doc/MATH.rst

```diff
@@ -0,0 +1,180 @@
+
+Core Concept
+''''''''''''
+
+One important fact about complex numbers is that one can think of the same
+complex number in two ways: either as a traditional two-dimensional vector or
+as a length and an angle (measured starting from the positive X-axis). When
+adding two complex numbers, they behave exactly like vectors.
+
+But in contrast to vectors, one can also multiply them together and get a new
+complex number. In this multiplication, their second interpretation becomes
+useful: multiplying two complex numbers means multiplying their lengths and
+adding their angles.
+
+In-Game Examples
+''''''''''''''''
+
+Coordinate System
+^^^^^^^^^^^^^^^^^
+
+Taisei's gameplay takes place entirely on a 2D plane, using an "X" and "Y"
+coordinate system.
+
+.. image:: images/taisei-xy-01.png
+   :width: 300pt
+
+Complex numbers in Taisei represent the "X-axis" (or "real") and "Y-axis" (or
+"imaginary"). This is not exactly a widely-adopted way of thinking about it,
+but as mentioned in *Core Concept*, it allows us to perform operations on the
+coordinate system that wouldn't otherwise be possible using vectors.
+
+Simple Movement
+^^^^^^^^^^^^^^^
+
+Lets take a few concrete examples within Taisei's code. For let's look at a
+piece of movement code for a fairy:
+
+.. code-block:: c
+
+   enemy->move = move_towards(VIEWPORT_W/2.0 + 200.0 * I, 0.035);
+
+The important piece here, for our purposes, is:
+
+.. code-block:: c
+
+   VIEWPORT_W/2.0 + 200.0 * I
+
+What this is doing is specifying a position on the "real" X-axis
+(``VIEWPORT_W/2.0``, which in Taisei means ``480.0 / 2.0`` or `240.0`), and
+then specifying a position on the "imaginary" Y-axis (``200.0 * I``). The ``I``
+here is the same ``i`` described in the videos and the above explanation.
+
+So what we're really looking at here is:
+
+.. code-block:: c
+
+   240.0 (real) + 200.0i (imaginary)
+
+Or "move 240 units on the (real) X-axis, and then 200 units on the (imaginary)
+Y-axis."
+
+This is what's called a **Cartesian Coordinate.** What the function
+``move_towards`` then does is make the enemy sprite/object move towards that
+point on the X/Y axis at a certain rate (defined by ``0.035``).
+
+Simple Danmaku
+^^^^^^^^^^^^^^
+
+Of course, there'd be no point in using complex numbers over a simpler X/Y
+system if it didn't provide significant advantages downstream. So let's look at
+a more complicated danmaku pattern to see why complex numbers are more
+effective for this role.
+
+.. code-block:: c
+
+   cmplx aim = cnormalize(global.plr.pos - enemy->pos);
+
+This ``aim`` variable could be passed to a ``move_towards`` function attached
+to a ``PROJECTILE`` object. The effect here would be the bullets moving
+directly towards the player in a straight line, even if the player moves around
+the screen, following the player as they move around in real-time (assuming)
+more than one bullet is fired.
```

Comment:
I think everything after "moving directly towards the player in a straight line" only adds confusion here. "Even if the player moves around the screen, following the player as they move around in real-time" sounds like you're describing some kind of homing behavior.